dotcom-rendering/src/components/Card/components/ImageWrapper.tsx

```diff
@@ -3,19 +3,31 @@ import { css } from '@emotion/react';
 import { between, from, until } from '@guardian/source/foundations';
 import { PlayIcon } from './PlayIcon';
 
-export type ImagePositionType = 'left' | 'top' | 'right' | 'bottom' | 'none';
+const imageFixedSize = {
+	tiny: 86,
+	small: 122.5,
+	medium: 125,
```

Comment:
These names feel a little awkward given that the sizes are now _so_ close (`small` was previously 97.5). I wonder if we could get away with making 122.5 the default fixed size everywhere in place of 125?